railties/lib/rails/dockerfile_helpers.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+require "active_support/core_ext/object/with_options"
+
+module Rails
+  module DockerfileHelpers
```

Comment:
I'm not sure of the best place for this module.  I want extension to be simple, so I would prefer to avoid an obscure nesting like `Rails::Command::DockerCommand::DockerfileHelpers`.